.pre-commit-config.yaml

```
@@ -20,6 +20,15 @@ repos:
         files: (package\.json|package-lock\.json)$
         always_run: false
         pass_filenames: false
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)